ComSemApp/models.py

```diff
@@ -33,6 +33,7 @@ def speaking_practice_audio_directory(directory, instance):
 # STUDENTS, TEACHERS, ADMINS
 
 class Student(models.Model):
+    id = models.AutoField(primary_key=True, serialize=False, verbose_name='ID')
```

Comment:
were these removed and then added back or are these being added for the first time?

Reply:
They're being added for the first time, they're auto-added by generated migrations and I added them for intellisense to work properly